### PR TITLE
Add canned text commands

### PR DESCRIPTION
**Pull request by PEZ**
Adding two commands for dealing with canned text, since it is a bit clunky to use replacers for that.

* **Paste Canned...**, brings up a menu with configured canned texts
* **Paste Text...**, keyboard shortcuts only, pastes a specific text

See the updated README for how to use.

Here's a VSIX for testing: https://output.circle-artifacts.com/output/job/7418564e-6798-4901-9b59-d42d6068b0cc/artifacts/0/tmp/artifacts/paste-replaced-1.1.11-canned-text-a91fabda.vsix

### CODE REVIEW

**Comment by KcZer0** [comment]
Hi, IMO, this shouldn't be put in this already fine extension, regex replacing text from clipboard. 
This overcomplicates this extension, so I'd suggest creating a new extension for this. 

I also don't really understand what "Canned" means.
A quick look in the readme, I assume it's predefined text to paste? 
Wouldn't that be the same as snippets? 

I can't really test it since I don't understand it, and don't have a use case for it. 
If it is predefined text, then snippets would be enough for me. 

Also, I'd really rather *not* learn a new syntax just to use an extension. 
If json is too limiting, then I'd prefer using javascript config.
Or other already popular config like yaml or toml, which is simple and easy to read. 

Lastly, it doesn't make sense to apply replacers to a predefined text.
If I were to use it, I would replace them before hand for faster paste (no waiting for replace, e.g. complex regex).

**Comment by AndrewRocky** [comment]
tl;dr - sounds difficult and not for my usecases

I've looked into updated README - it does look like a nice universal solution for many problems. Just like vim/emacs/etc editors are good because of their extensibility... but are quite difficult to learn.
I see that this extension might become a nice tool for video tutorial makers - having both snippets and humanlike typing in a single package.

The problems is... I don't really use any of these features - I only wanted a regex-based replace-paste tool for keybinding. Though I don't mind having them in this extension. 

Sorry since I can't really help with testing this part of extension.

**Comment by PEZ** [comment]
Thanks, both of you!

Yes canned means prepared here. And yes, very similar to snippets. Although you can't make snippets paste simulating writing, and you can't use them in the terminal (e.g.).

I plan to allow for configuring the canned texts in regular settings.json. But I need this for a presentation I am doing soon and want to be able to actually read the code where I define it. Compare:

```clojure
 {:name "Joyride: Are you a Joyrider?"
  :text (p/let [choice (vscode/window.showInformationMessage "Be a Joyrider 🎸" "Yes" "Of course!")]
          (if choice
            (.appendLine (joyride/output-channel) (str "You choose: " choice " 🎉"))
            (.appendLine (joyride/output-channel) "You just closed it? 😭")))}
```

```json
  {
      "name": "Joyride: Are you a Joyrider?",
      "replacements": "(p/let [choice (vscode/window.showInformationMessage \"Be a Joyrider 🎸\" \"Yes\" \"Of course!\")]\n  (if choice\n    (.appendLine (joyride/output-channel) (str \"You choose: \" choice \" 🎉\"))\n    (.appendLine (joyride/output-channel) \"You just closed it? 😭\")))"
  },
```

I hear you both about not needing this feature and not understanding the use case to be able to test it. Still, if you have the time, you can test that it doesn't break your existing uses. I've been refactoring stuff a bit. I'm not too worried, though, I've been testing it quite a lot myself.
